src/apps/deskflow-gui/MainWindow.h

```diff
@@ -184,16 +184,18 @@ class MainWindow : public QMainWindow
    */
   QString trustedFingerprintDb();
 
+  inline static const auto m_nameRegEx = QRegularExpression(QStringLiteral("^[\\w\\-_\\.]{0,255}$"));
+
   // Generate prints if they are missing
   // Returns true if successful
   bool regenerateLocalFingerprints();
+  inline static const auto m_guiSocketName = QStringLiteral("deskflow-gui");
```

Comment:
Please group static member variables together. The `m_nameRegEx` var needs to be moved down under the member functions. We should group it with `m_guiSocketName`.

Also, lack of space makes `regenerateLocalFingerprints` and `m_guiSocketName` appear related, which they are not.

```suggestion
  // Generate prints if they are missing
  // Returns true if successful
  bool regenerateLocalFingerprints();
  
  inline static const auto m_nameRegEx = QRegularExpression(QStringLiteral("^[\\w\\-_\\.]{0,255}$"));
  inline static const auto m_guiSocketName = QStringLiteral("deskflow-gui");
```